packages/backend/src/apps/pipedrive/dynamic-data/list-persons/index.ts

```diff
@@ -23,6 +23,10 @@ export default {
       );
       params.start = data.additional_data?.pagination?.next_start;
 
+      if (!data?.data) {
+        return { data: [] };
+      }
+
```

Comment:
We already have an empty state `persons` variable in place. Making the if condition ahead fail-safe should be sufficient to fix the problem rather than introducing another logical branch as early exit.